asciidocs/modules.adoc

```diff
@@ -146,21 +146,21 @@ executor >  local (6)
 [c0/115bd6] process > SPLITLETTERS_two (1)   [100%] 1 of 1 ✔
 [09/f9072d] process > CONVERTTOUPPER_two (2) [100%] 2 of 2 ✔
 WORLD!
-HELLO 
+HELLO
 WORLD!
-HELLO 
+HELLO
 ----
 ====
 
-NOTE: You can store each process in separate files within separate sub-folders or combined in one big file (both are valid). For your own projects, it can be helpful to see examples on public repos. e.g. Seqera RNA-Seq tutorial 
-https://github.com/seqeralabs/rnaseq-nf/tree/master/modules[here] or within nf-cores projects  https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules[here].
+NOTE: You can store each process in separate files within separate sub-folders or combined in one big file (both are valid). For your own projects, it can be helpful to see examples on public repos. e.g. Seqera RNA-Seq tutorial
+https://github.com/seqeralabs/rnaseq-nf/tree/master/modules[here] or within nf-core projects  https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules[here].
```

Comment:
```suggestion
https://github.com/seqeralabs/rnaseq-nf/tree/master/modules[here] or within nf-core modules project https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules[here].
```

Reply:
Small suggestion regarding the name of Github repo

Reply:
The link isn't to the modules project though, it's to the rnaseq pipeline. I think this is about file organisation right? So the link looks good but I agree that the text is a bit vague.

```suggestion
https://github.com/seqeralabs/rnaseq-nf/tree/master/modules[here] or within nf-core pipelines, such as https://github.com/nf-core/rnaseq/tree/master/modules/nf-core/modules[nf-core/rnaseq].
```

Reply:
Ah, then the `/modules` got me - good catch Phil!